app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,119 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.enums.HttpMethod;
+import com.codesoom.assignment.enums.HttpStatus;
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DemoHttpHandler implements HttpHandler {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private List<Task> tasks = new ArrayList<>();
+    private Long id = 1L;
+
+    public DemoHttpHandler() {
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        String path = exchange.getRequestURI().getPath();
+
+        System.out.println(
+                String.format("%s %s Something requests...",
+                        method,
+                        path
+                )
+        );
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader((inputStream)))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        int responseCode = HttpStatus.OK.code();
```

Comment:
여기에서 `int` 타입을 사용할 필요가 있을까요? 다음과 같이 해보면 어떤 장점이 있을까요?
```suggestion
        HttpStatus responseCode = HttpStatus.OK;
```

Reply:
흠.. 생각해볼 부분인 것 같습니다.
enum에 너무 익숙하지 않아서 말씀해주신 부분은 생각해보지 못했습니다. 사용법만 ctrl c v 만 했네요 ㅎㅎ..